tmt/steps/discover/fmf.py

```diff
@@ -430,30 +442,30 @@ def assert_git_url(plan_name: Optional[str] = None) -> None:
                             shutil.copytree(
                                 git_root, self.testdir, symlinks=True)
 
-            # Initialize or remove fmf root
-            if dist_git_init:
-                if not dist_git_extract:
-                    dist_git_extract = sourcedir
-                if not self.opt('dry'):
-                    fmf.Tree.init(dist_git_extract)
-            elif dist_git_remove_fmf_root:
-                if not self.opt('dry'):
-                    shutil.rmtree(os.path.join(
-                        dist_git_extract or top_fmf_root, '.fmf'))
-                if not dist_git_extract:
-                    dist_git_extract = sourcedir
-
-            # Now can safely default to top_fmf_root
+            # Set dist_git_extract if not set yet
             if not dist_git_extract:
-                dist_git_extract = top_fmf_root
+                dist_git_extract = [{
+                    "src": sourcedir,
+                    "dest": '.',
+                }]
 
             # Now copy dist_git_extract into tests
             if not self.opt('dry'):
-                tmt.utils.copytree(
-                    dist_git_extract,
-                    self.testdir,
-                    symlinks=True,
-                    dirs_exist_ok=True)
+                # Copy all requested paths
+                for path in dist_git_extract:
+                    try:
+                        tmt.utils.copytree(
+                            path['src'],
+                            self.testdir.joinpath(path['dest']),
```

Comment:
`self.testdir / path['dest']` (or `dst` as suggested above).

Reply:
Oh `/` is an alias of `joinpath`? Just making sure there is no weird things that can happen since the filereaders/loaders are overwiriding `joinpath` and `iterdir` when they are implementing custom readers like zip reader, lookaside, etc.

Reply:
Yep, `/` is an alias for `joinpath`, it works in the same way by the magic of dunder methods.